src/util/pointInFreehandROI.js

```diff
@@ -0,0 +1,66 @@
+// JPETTS - Calculates if "point" is inside the polygon defined by dataHandles by
```

Comment:
Can you switch this to a comment on the actual function, rather than in the file?